content/getstarted/02-item.md

```diff
@@ -1,56 +1,193 @@
+deployment.apps/cdi-uploadproxy created
+$
+```
 
-# Create an APP that uses the created volume
-cluster/kubectl.sh create -f csi/app.yml
+At this point, we have a fully functioning OpenShift cluster with KubeVirt along side Ember-CSI backed by a Ceph backend. We have also created a namespace/project called `sample-project` which we will use to deploy a VM using KubeVirt. Before we proceed further, source the `env.sh` file which helps create useful aliases for us to use.
+
+```shell
+$ source tools/env.sh
+$ k create -f examples/cirros-pvc.yml
+persistentvolumeclaim/cirros-pvc created
+$ k get pvc
+NAME         STATUS    VOLUME                 CAPACITY   ACCESS MODES   STORAGECLASS                    AGE
+cirros-pvc   Bound     pvc-b19271605d2611e9   3Gi        RWO            io.ember-csi.external-ceph-sc   3s
+$
 ```
+If the `status` above says `BOUND`, we will then check to see if the CDI Importer pod has finished importing. Once done, the pod (job) vanishes, we will proceed to creating the VM.
+$ k get pod
+NAME                            READY     STATUS    RESTARTS   AGE
+pod/importer-cirros-pvc-8lr4k   1/1       Running   0          36s
+$ k logs pod/importer-cirros-pvc-8lr4k
+I0412 15:30:01.911266       1 importer.go:45] Starting importer
+I0412 15:30:01.920943       1 importer.go:58] begin import process
+I0412 15:30:01.920993       1 importer.go:82] begin import process
+I0412 15:30:01.921027       1 dataStream.go:293] copying "https://download.cirros-cloud.net/0.4.0/cirros-0.4.0-x86_64-disk.img" to "/data/disk.img"...
+I0412 15:30:02.407774       1 prlimit.go:107] ExecWithLimits qemu-img, [info --output=json https://download.cirros-cloud.net/0.4.0/cirros-0.4.0-x86_64-disk.img]
+I0412 15:30:03.310515       1 prlimit.go:107] ExecWithLimits qemu-img, [convert -p -f qcow2 -O raw json: {"file.driver": "https", "file.url": "https://download.cirros-cloud.net/0.4.0/cirros-0.4.0-x86_64-disk.img", "file.timeout": 3600} /data/disk.img]
+I0412 15:30:03.320607       1 qemu.go:189] 0.00
+I0412 15:30:04.207629       1 qemu.go:189] 1.19
+I0412 15:30:04.208937       1 qemu.go:189] 2.38
+I0412 15:30:04.265587       1 qemu.go:189] 3.57
+...
+I0412 15:30:09.946197       1 qemu.go:189] 99.21
+$
+
+### Creating and Managing a Virtual Machine
+
+Lets begin by creating a VM template.
 
-We can send GRPC commands to the CSI driver using the deployed csc container.
+```shell
+$ k create -f examples/cirros-vm.yml
+virtualmachine.kubevirt.io/cirros-vm created
+$
+```
 
-For example to list volumes you can do:
+Inspect whether the VM is created and running successfully
 
+```
+$ k get vms
+NAME        AGE       RUNNING   VOLUME
+cirros-vm   3s        true      
+$
+```
+
+The running VM can be accessed using the console using the command:
+```shell
+$ v console cirros-vm
+Successfully connected to cirros-vm console. The escape sequence is ^]
+[    0.000000] Initializing cgroup subsys cpuset
+[    0.000000] Initializing cgroup subsys cpu
+
+... 
```

Comment:
In my case this took a while as it kept doing:

```
udhcpc (v1.23.2) started
Sending discover...
Sending select for 10.128.0.67...
Lease of 10.128.0.67 obtained, lease time 86313600
route: SIOCADDRT: File exists
WARN: failed: route add -net "0.0.0.0/0" gw "10.128.0.1"
checking http://169.254.169.254/2009-04-04/instance-id
failed 1/20: up 60.13. request failed
failed 2/20: up 72.24. request failed

[ ... ]

failed to read iid from metadata. tried 20
failed to get instance-id of datasource

[ ... ]
```

But in the end it started successfully.  If this is normal behavior we should somehow show it here to avoid scaring people.